README.md

```diff
@@ -101,7 +101,7 @@ $ zip-it-and-ship-it --help
 @netlify/zip-it-and-ship-it: Zip lambda functions and their dependencies for deployment
 
 Usage: zip-it-and-ship-it [source] [destination] {options}
-    --zip-go, -g          zip go binaries (default: false)
+    --skip-go             do not zip go binaries (default: true)
```

Comment:
a boolean flag with a default of `true` does not make sense to me

Reply:
You're right!
I just fixed it in #91, could you check it out? Thanks!